starter-code/css/main.css

```diff
@@ -6,3 +6,133 @@ Green: #00B172
 White: #FFF
 
 */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Helvetica,Arial,sans-serif;
+}
+
+.nav-bar {
+  background-color: white;
+  height: 75px;
+  width: 100%;
+  padding: 15px 10px 5px 10px;
+
+}
+ .nav-bar > ul {
+    width: 1200px;
+    margin: 0 auto;
+ }
+
+ .nav-bar li {
+   text-align: right;
+   float: right;
+   padding: 20px 10px 14px 10px;
+   list-style-type: none;
+   font-size: 20px; 
+   margin:0 0 10px 0;  
+   font-size: 20px;
+   font-weight: 100; 
+ }
+
+ .nav-bar a {
+   text-decoration: none;
+   color: black;
+ }
+
+ .music {
+   text-align: center;
+   height: 800px;
+   background: url("../images/landing.jpg");
+   background-size: cover;
+   background-repeat: no-repeat;  
+   color: white;
+   text-align: center;
+ }
+
+ .music > h1 {
+   position: relative;
+   padding-top: 17%;
+   padding-bottom: 20px;
+   line-height: 70px;
+   font-size: 4.5em;
+ }
+
+ .music > p {
+   font-size: 30px;
+   font-weight: 200;
+ }
+
+ .top-logo {
+  float: left;
+  height: 72px;
+  width: 260px;
+
+ }
+
+.container > h2 {
+  text-align: center;
+  font-size: 40px;
+  font-weight: 400;
+  
+}
+
+.container hr {
+  margin-left: 329px;
+  width: 334px;
+  border-bottom: 2px solid green;
+}
+
+.container {
+  height: 500px;
+  padding: 20px 10px;
+  margin: 5px 95px 0 95px;
+}
+
+.content > .sound {
+  height:40%;
+  width: 40%;
+  padding-top: 70px;
+}
+
+.content {
+text-align: center;
+padding: 70px 0; 
+
+}
+
+.green-box {
+  background-color: #00B172;
+	margin:0  50px;
+  height: 800px;
+  width: 1000px;
+  color: #FFF;
+  padding: 0 20px;
+}
+
+.left {
```

Comment:
I dont like this class name.... bad choice calling it "left" when all you did here was change the font.  You could have done something like: 
```css
.col-left {
   float: left
}
```
And left that as a reusable class, and to target the font then you could have just created some other class name that was more specific.